client/api.go

```diff
@@ -163,330 +163,409 @@ import (
+	v2moneymanagementoutboundsetupintent "github.com/stripe/stripe-go/v81/v2/moneymanagement/outboundsetupintent"
+	v2moneymanagementoutboundtransfer "github.com/stripe/stripe-go/v81/v2/moneymanagement/outboundtransfer"
+	v2moneymanagementpayoutmethod "github.com/stripe/stripe-go/v81/v2/moneymanagement/payoutmethod"
+	v2moneymanagementpayoutmethodsbankaccountspec "github.com/stripe/stripe-go/v81/v2/moneymanagement/payoutmethodsbankaccountspec"
+	v2moneymanagementreceivedcredit "github.com/stripe/stripe-go/v81/v2/moneymanagement/receivedcredit"
+	v2moneymanagementreceiveddebit "github.com/stripe/stripe-go/v81/v2/moneymanagement/receiveddebit"
+	v2moneymanagementtransaction "github.com/stripe/stripe-go/v81/v2/moneymanagement/transaction"
+	v2moneymanagementtransactionentry "github.com/stripe/stripe-go/v81/v2/moneymanagement/transactionentry"
+	v2testhelpersfinancialaddress "github.com/stripe/stripe-go/v81/v2/testhelpers/financialaddress"
 	"github.com/stripe/stripe-go/v81/webhookendpoint"
 )
 
 // API is the Stripe client. It contains all the different resources available.
 type API struct {
-	// AccountLinks is the client used to invoke /account_links APIs.
+	// AccountLinks is the client used to invoke /v1/account_links APIs.
 	AccountLinks *accountlink.Client
-	// AccountNotices is the client used to invoke /account_notices APIs.
+	// AccountNotices is the client used to invoke /v1/account_notices APIs.
 	AccountNotices *accountnotice.Client
-	// Accounts is the client used to invoke /accounts APIs.
+	// Accounts is the client used to invoke /v1/accounts APIs.
 	Accounts *account.Client
-	// AccountSessions is the client used to invoke /account_sessions APIs.
+	// AccountSessions is the client used to invoke /v1/account_sessions APIs.
 	AccountSessions *accountsession.Client
-	// ApplePayDomains is the client used to invoke /apple_pay/domains APIs.
+	// ApplePayDomains is the client used to invoke /v1/apple_pay/domains APIs.
 	ApplePayDomains *applepaydomain.Client
-	// ApplicationFees is the client used to invoke /application_fees APIs.
+	// ApplicationFees is the client used to invoke /v1/application_fees APIs.
 	ApplicationFees *applicationfee.Client
-	// AppsSecrets is the client used to invoke /apps/secrets APIs.
+	// AppsSecrets is the client used to invoke /v1/apps/secrets APIs.
 	AppsSecrets *appssecret.Client
-	// Balance is the client used to invoke /balance APIs.
+	// Balance is the client used to invoke /v1/balance APIs.
 	Balance *balance.Client
-	// BalanceTransactions is the client used to invoke /balance_transactions APIs.
+	// BalanceTransactions is the client used to invoke /v1/balance_transactions APIs.
 	BalanceTransactions *balancetransaction.Client
-	// BankAccounts is the client used to invoke bankaccount related APIs.
+	// BankAccounts is the client used to invoke /v1/accounts/{account}/external_accounts APIs.
```

Comment:
haha - nice! Was this manually done or generated?

Reply:
What I meant was that we replaced a generic statement like "blah related APIs" with the actual link - that is good